global_const.js

```diff
@@ -0,0 +1,11 @@
+// Property to determine if the importer should be run against the Pelias ES index that contains the polygon geo-shape data.
+global.geo_shape_polygon=true;
+
+// Validation for constants
+if(global.geo_shape_polygon === true || global.geo_shape_polygon === false){
+  console.log('\n Running the importer against the ES index that contains the polygon geo-shape data: ' + global.geo_shape_polygon + '.');
+}else{
+  console.error('The global.geo_shape_polygon property in the global_const.js file has not been correctly configured.\n');
+  console.error('\nPlease set this value to true or false and try again.');
+  process.exit(1);
+}
```

Comment:
Please remove this file completely. If you'd like to make whether or not shapes are imported configurable (a great idea), use [pelias/config](https://github.com/pelias/config) to do so.

A great example of how that works is in the downloader, where the URL base is configurable:
https://github.com/pelias/whosonfirst/blob/839f5b71e8e72ae75a362e8df7343fa1dfb0f5c7/utils/download_sqlite_all.js#L13

In this case, to fit with our other properties, you can check if `imports.whosonfirst.importShapes` is set, and default to _not_ importing shapes for now. Let me know if that makes sense or you need more examples.